tachyon/base/json/rapidjson_util.h

```diff
@@ -268,6 +268,37 @@ class RapidJsonValueConverter<std::vector<T>> {
   }
 };
 
+template <typename T>
+class RapidJsonValueConverter<std::pmr::vector<T>> {
```

Comment:
Please include `<memory_resource>`
Please also check the files below:
`tachyon/math/polynomials/univariate/univariate_dense_polynomial_unittest.cc`
`tachyon/math/polynomials/univariate/univariate_evaluation_domain_unittest.cc`
`tachyon/math/polynomials/univariate/univariate_evaluations_unittest.cc`
`tachyon/zk/plonk/constraint_system/selector_compressor_unittest.cc`
`tachyon/zk/plonk/expressions/compress_expression_unittest.cc`
`tachyon/zk/plonk/permutation/unpermuted_table_unittest.cc`
`vendors/circom/prover_main.cc`

Reply:
Generally, I didn't include in the headers which doesn't include `<vector>` in the first place.

Reply:
I only added to rapidjson. Others rely on transitive include in the first place so I didn't add include.